app/controllers/pages_controller.rb

```diff
@@ -9,4 +9,28 @@ def sign_in_development
     sign_in :user, user, bypass: true
     redirect_to params[:redirect_to] || root_path
   end
+
+  def contact
+    @contact_email = ContactForm.new(
+      email: current_user.email
+    )
+  end
+
+  def submit_contact_form
+    @contact_email = ContactForm.new(contact_form_params)
+    if @contact_email.save
+      UserMailer.contact(@contact_email).deliver
+      flash.now[:notice] = "Your email sent"
+      @contact_email.text = ""
+    end
+    render :contact, status: :unprocessable_entity
```

Comment:
Why we have same status on both valid and valid case ?

Reply:
Should not be the same. If is invalid it triggers errors: 
 ```bash
 render :contact, status: :unprocessable_entity
 ```
with: 
```bash
<% if @contact_form.errors.any? %>
          <div class="text-red-500">
            <%= @contact_form.errors.full_messages.to_sentence.capitalize %>
          </div>
        <% end %>
```
in simple_form.